src/ast/ast_build_ar_exp.c

```diff
@@ -387,14 +387,17 @@ static AR_ExpNode *_AR_EXP_FromExpression(const cypher_astnode_t *expr) {
 		return _AR_ExpFromNamedPath(expr);
 	} else if(type == CYPHER_AST_NODE_PATTERN || type == CYPHER_AST_REL_PATTERN) {
 		return _AR_ExpNodeFromGraphEntity(expr);
+	} else if(type == CYPHER_AST_PARAMETER) {
+		AST *ast = QueryCtx_GetAST();
+		const cypher_astnode_t *param_value = cypher_astnode_get_annotation(ast->params_ctx, expr);
+		return _AR_EXP_FromExpression(param_value);
```

Comment:
The change suggested at https://github.com/RedisGraph/RedisGraph/pull/731/files#r344807334 would allow us to replace this call, though I might be overlooking consequences.

Reply:
changed